pimd/pim_bsm.c

```diff
@@ -1223,9 +1219,8 @@ int pim_bsm_process(struct interface *ifp, struct ip *ip_hdr, uint8_t *buf,
 	/* BSM Packet acceptance validation */
 	pim_ifp = ifp->info;
 	if (!pim_ifp) {
-		if (PIM_DEBUG_BSM)
-			zlog_debug("%s: multicast not enabled on interface %s",
-				   __func__, ifp->name);
+		zlog_warn("%s: multicast not enabled on interface %s", __func__,
```

Comment:
Are we going to fill the logs up with this warning when pim is not configured on an interface that we haven't turned on multicast yet for?  Seems like there was a reason this was a debug.